packaging/install.cmake

```diff
@@ -67,60 +67,64 @@ endif()
 
 if(UNIX)
 
-  # replace MPIEXEC MPIEXEC_NUMPROC_FLAG and MPI_IMPL KHIOPS_MPI_EXTRA_FLAG ADDITIONAL_EN_VAR
+  # Set khiops and khiops_coclustering paths according to the environment (conda, fedora, etc)
+  if(IS_CONDA)
+    set(KHIOPS_PATH "$(get_script_dir)")
+    set(KHIOPS_COCLUSTERING_PATH "$(get_script_dir)")
+    set(GET_PROC_NUMBER_PATH "$(get_script_dir)")
+  else()
+    if(IS_FEDORA_LIKE)
+      set(KHIOPS_PATH "${MPI_BIN}/khiops/")
+    else()
+      set(KHIOPS_PATH "/usr/bin/")
+    endif(IS_FEDORA_LIKE)
+    set(KHIOPS_COCLUSTERING_PATH "/usr/bin/")
+    set(GET_PROC_NUMBER_PATH "/usr/bin/")
+  endif(IS_CONDA)
+
+  # replace MPIEXEC MPIEXEC_NUMPROC_FLAG and MPI_IMPL KHIOPS_MPI_EXTRA_FLAG ADDITIONAL_ENV_VAR
   if("${MPI_IMPL}" STREQUAL "openmpi")
     set(KHIOPS_MPI_EXTRA_FLAG "--allow-run-as-root --quiet")
-    set(ADDITIONAL_EN_VAR "export OMPI_MCA_btl_vader_single_copy_mechanism=none # issue on docker")
+    set(ADDITIONAL_ENV_VAR "export OMPI_MCA_btl_vader_single_copy_mechanism=none # issue on docker")
     if(IS_FEDORA_LIKE)
-      set(ADDITIONAL_EN_VAR "${ADDITIONAL_EN_VAR}\nexport PSM3_DEVICES=self # issue one rocky linux")
+      set(ADDITIONAL_ENV_VAR "${ADDITIONAL_ENV_VAR}\nexport PSM3_DEVICES=self # issue one rocky linux")
     endif()
+    set(MPIEXEC_HOSTFILE_FLAG "--hostfile")
+    set(EXPORT_ENV_VARIABLES
+        "# Export environment variables to MPI processes
+    for line in $(env | grep -E '^(KHIOPS|Khiops|AWS_|S3_|GOOGLE_)'); do
+        name=\${line%%=*}
+        MPI_EXTRA_FLAGS=\"\${MPI_EXTRA_FLAGS} -x \${name}\"
+    done")
+  elseif("${MPI_IMPL}" STREQUAL "mpich")
+    set(MPIEXEC_HOSTFILE_FLAG "--hostfile")
```

Comment:
`@MPIEXEC_HOSTFILE_FLAG@` will equal `--hostfile` for both OpenMPI and MPICH as far as I can see; then, why not factor this setting out so that we don't need to repeat it; something like:

```cmake
IF(NOT "${MPI_IMPL}" STREQUAL "")
  set(MPIEXEC_HOSTFILE_FLAG "--hostfile")
endif()
```
?
Also, in the commit message for commit https://github.com/KhiopsML/khiops/pull/343/commits/ee1887378b4a6e07e130f4b48efb6cc1e7b6b1c6 it is stated that "[...] the hostfile keyword is different for openmpi and mpich": where is the difference?